### DISCUSSION TITLE
Botw Question

### DISCUSSION THREAD

**Discussion by ARG14** [skyline-emu/skyline#1494] [General]
Just to get my hopes up, do you have any idea when this great game will come out?
And be playable on Skyline

**Comment by Swizzlefowl** [comment]
It came out a long time ago on the switch, and from what I have heard it's very playable.

**Reply by guranu** [reply]
it only goes to the intro.

**Reply by ARG14** [reply]
I just got the game but it won't even boot why?

**Comment by Swizzlefowl** [comment]
Just to clarify I meant it's playable on the switch.
